.circleci/config.yml

```diff
@@ -1,7 +1,33 @@
 version: 2
 jobs:
+  kafka-010:
+    working_directory: /go/src/github.com/segmentio/kafka-go
+    environment:
+      KAFKA_VERSION: "0.10.1"
```

Comment:
should be 0.10.1.0 ?

Reply:
Do we need to go all the way out to the 4th number?  I thought protocol changes couldn't happen at that release level.

Reply:
You're correct ! :)